### ISSUE TITLE
simBEV hangs for LR_Zentr and larger number of cars

### ISSUE THREAD

**Issue by nesnoj** [rl-institut/simbev#18]
Reproduce by
1. use current dev or master
2. set `regio_type = LR_Zentr`
3. in section `[rampup_ev]` set all BEVs to 100
4. run

You'll see
```
Writing to res/2021-05-04_162436_simbev_run
2% bev_luxury 6 / 100
```
and it seems to hang... (always at bev_luxury)
Aborting reveals that it's something with the random stuff here:
```
Traceback (most recent call last):
  File "main_simbev.py", line 211, in <module>
    work_charging_capacity) = simbevMiD.availability(
  File "/home/nesnoj/git-repos/rl-institut/simbev/simbev/simbevMiD.py", line 313, in availability
    speed = rng.choice(
  File "_generator.pyx", line 864, in numpy.random._generator.Generator.choice
  File "/home/nesnoj/virtualenvs/simbev/lib/python3.8/site-packages/numpy/core/_asarray.py", line 16, in asarray
    @set_module('numpy')
KeyboardInterrupt
```
Don't have this problem in other regions..

@timroepcke Can you help?

**Comment by nesnoj** [comment]
> (always at bev_luxury)

Update: Also occurs when generating other car types.. but (until now) always in LR_Zentr

**Comment by nesnoj** [comment]
Fixed in 1569829c8ab4baeef8888c3e3b90742e1534f04f, thank you @timroepcke !
